flex/interactive/sdk/examples/python/vertex_edge_example.py

```diff
@@ -0,0 +1,211 @@
+import gs_interactive
+from gs_interactive.models.vertex_request import VertexRequest
+from gs_interactive.models.edge_request import EdgeRequest
+from gs_interactive.models.model_property import ModelProperty
+from gs_interactive.models.vertex_edge_request import VertexEdgeRequest
+from gs_interactive.rest import ApiException
+import argparse
```

Comment:
Maybe we should move the implementation into `DefaultSession` in `session.py`, implementing the abstract methods?

Reply:
I've moved the code accordingly.